### DISCUSSION TITLE
Warning calculate-size

### DISCUSSION THREAD

**Discussion by bourhanmamod** [reaviz/reaflow#163] [Q&A]
Hello,
Whenever I use this module, I have this warning: 
Failed to parse source map from 'mypath/node_modules/calculate-size/src/index.ts' file: Error: ENOENT: no such file or directory, open 'mypath/node_modules/calculate-size/src/index.ts'

Can anyone tell me how do I correct it?

Thanks in advance

**Comment by amcdnl** [comment]
I haven't seen this. Is it actually failing or just warning?

**Reply by bourhanmamod** [reply]
It's just a Warning

**Reply by bourhanmamod** [reply]
![Capture d’écran 2022-06-01 152511](https://user-images.githubusercontent.com/44088800/171415401-6532b7af-107c-4bdc-bbd8-6f85c43a2be4.png)
